scripts/dts/edtlib.py

```diff
@@ -755,6 +759,11 @@ def name(self):
         "See the class docstring"
         return self._node.name
 
+    @property
+    def is_device(self):
+        "See the class docstring"
+        return self._binding and self._binding.get('is-device', False)
```

Comment:
Just a personal quirk, but make it `"is-device"` instead of `'is-device'` for consistency with the rest of the code.